[2024-05-10 01:50:55][logger.py] INFO: Удалены лишние лог-файлы: 0
[2024-05-10 01:50:55][logger.py] INFO: Текущий лог-файл: 2024 05-10 01.50.55.log
[2024-05-10 01:51:02][logger.py] INFO: Retrieved user 6034429696's *: (6034429696, 'cardinal', None, 'ru', 'cardinaler', '2024-05-10 01:31:24.145166', '', 0, '1744691089', 0, '', 0)
[2024-05-10 01:51:11][logger.py] INFO: Получены данные callback - {'func': 'open_category', 'category': '49654'}
[2024-05-10 01:51:12][logger.py] INFO: Получены данные callback - {'func': 'open_product', 'cat': '49654', 'prod': '62536'}
[2024-05-10 01:51:12][logger.py] INFO: Лота нет в изобранном
[2024-05-10 01:51:14][logger.py] INFO: Получены данные callback - {'func': 'buy', 'cat': '49654', 'prod': '62536', 'count': '3'}
[2024-05-10 01:51:15][logger.py] INFO: Получены данные callback - {'func': 'buy', 'cat': '49654', 'prod': '62536', 'count': '2'}
[2024-05-10 01:51:15][logger.py] INFO: Retrieved user 6034429696's balance: (0,)
[2024-05-10 01:51:15][logger.py] INFO: Retrieved user 6034429696's balance: (0,)
[2024-05-10 01:51:15][logger.py] INFO: Сохранил новый заказ в файл JSON
[2024-05-10 01:51:15][logger.py] INFO: Retrieved user 6034429696's balance: (0,)
[2024-05-10 01:51:15][logger.py] INFO: Retrieved user 6034429696's balance: (0,)
[2024-05-10 01:51:15][logger.py] INFO: Updated user 6034429696's balance to 0
[2024-05-10 01:51:15][logger.py] INFO: Added bonus 0 to user 6034429696
[2024-05-10 01:51:16][logger.py] INFO: Получены данные callback - {'func': 'check_status_order', 'order': 'fSp4EE9cni'}
[2024-05-10 01:51:16][logger.py] INFO: Обновил кол-во товаров. Номер товара - 62536. Номер - категории - 49654
[2024-05-10 01:51:16][logger.py] INFO: Отправил файл с товарами юзеру - 6034429696. Номер товара - 62536. Номер - категории - 49654
[2024-05-10 01:51:16][logger.py] ERROR: Пользователь 6034429696 оплатил заказ fSp4EE9cni
[2024-05-10 01:51:16][logger.py] INFO: Retrieved user 6034429696's from_referal: ('1744691089',)
[2024-05-10 01:51:16][logger.py] CRITICAL: Ошибка при обработке callback - unsupported operand type(s) for /: 'str' and 'int'
[2024-05-10 01:52:06][logger.py] INFO: Получены данные callback - {'func': 'open_category', 'category': '49654'}
[2024-05-10 01:52:08][logger.py] INFO: Получены данные callback - {'func': 'open_product', 'cat': '49654', 'prod': '62536'}
[2024-05-10 01:52:08][logger.py] INFO: Лота нет в изобранном
[2024-05-10 01:52:14][logger.py] INFO: Получены данные callback - {'func': 'open_category', 'cat': '49654'}
[2024-05-10 01:52:15][logger.py] INFO: Получены данные callback - {'func': 'open_product', 'cat': '49654', 'prod': '97010'}
[2024-05-10 01:52:15][logger.py] INFO: Лота нет в изобранном
